libs/types/codegen.yml

```diff
@@ -9,21 +9,21 @@ config:
 generates:
   libs/types/src/__generated__/types.ts:
     - typescript
-  libs/**/__generated__:
+  libs/**:
```

Comment:
☝🏻☝🏻☝🏻☝🏻☝🏻 

suggestion: How about enforcing the naming convention? Having it set to:

```yml
config:
  preResolveTypes: true
  namingConvention:
    enumValues: keep
    typeNames: 'change-case-all#pascalCase'
```

will deal with inconsistancies such as: `PartyliquidityProvisionsArgs` 👉🏻 `PartyLiquidityProvisionsArgs` or `QueryoracleSpecsConnectionArgs` 👉🏻  `QueryOracleSpecsConnectionArgs`

Reply:
to me it looks like that is the default setting (https://www.the-guild.dev/graphql/codegen/docs/config-reference/naming-convention):
<img width="570" alt="Screenshot 2022-09-08 at 18 52 14" src="https://user-images.githubusercontent.com/105208209/189180585-a428e2ce-edbe-4138-9fb3-4b6b07a442e5.png">